pyvista/demos/demos.py

```diff
@@ -5,7 +5,7 @@
 
 
 def glyphs(grid_sz=3, **kwargs):
-    """Plot several parametric supertoroids using VTK's glyph table functionality."""
+    """Create several parametric supertoroids using VTK's glyph table functionality."""
     n = 10
     values = np.arange(n)  # values for scalars to look up glyphs by
```

Comment:
We don't even need to pass this explicitly anymore, and the glyph table option will generate the same `values`. We might still keep it for educational purposes, perhaps?

Reply:
I'll be using this within another project/branch, which is partially why I updated the logo anyway.